tests/unit/entityInfo.spec.ts

```diff
@@ -53,58 +86,119 @@ describe.skip('Entity Info component with an NR', () => {
     const crumb2 = breadcrumbs.at(2)
     const crumb3 = breadcrumbs.at(4)
 
-    // TODO: add test for "Staff Dashboard" when user is staff
-    expect(crumb1.text()).toStrictEqual('Manage Businesses Dashboard')
+    expect(crumb1.text()).toStrictEqual('Staff Dashboard')
     expect(divider.text()).toStrictEqual('>')
-    expect(crumb2.text()).toStrictEqual('Xyz Ltd.')
-    expect(crumb3.text()).toStrictEqual('BC Benefit Company Incorporation Application')
+    expect(crumb2.text()).toStrictEqual('My Mock Name Inc.')
+    expect(crumb3.text()).toStrictEqual('Correction - Incorporation Application')
   })
-})
 
-describe.skip('Entity Info component without an NR', () => {
-  let wrapper: any
-  let store: any
-
-  beforeEach(() => {
-    store = getVuexStore()
+  it('renders the business name and numbers', async () => {
+    expect(wrapper.vm.$el.querySelector('#entity-legal-name').textContent)
```

Comment:
I would prefer to see us move to the new wrapper format here, eg,
`wrapper.find('#entity-legal-name').text()`.

(Otherwise we're reaching in to the element and calling the JS query function directly.)

Reply:
For sure, no problems there. I should have done this initially. Updated in next commit.